### PR TITLE
Fix UI in settings in change password when in dark theme

### LINKED ISSUE #1970: Fix UI in settings in change password when in dark theme
**Actual Behaviour**
The labels Current Password, New Password, Verify Password and are in black text on a dark background.

**Expected Behaviour**
The labels Current Password, New Password, Verify Password must be white.

**Screenshots of the issue**
screenshot 2019-01-21 at 8 01 36 pm

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #1970 
Changes: Fix UI in settings in change password when in dark theme

Demo Link: https://pr-1971-fossasia-susi-web-chat.surge.sh

Screenshots for the change: 
screenshot 2019-01-21 at 10 47 58 pm

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@fragm3 themeBackgroundColor is present inside the class and the labelStyle shall change with the background that is why I put the styles inside the class. I can keep the styles outside and just keep the label style inside if that is a better approach.

**Comment by AakashMallik** [comment]
Keep the styles out, only the property dependent on the theme value can be altered with the component. 
{{... xyzStyle, background: props. theme === 'dark'? a : b}}  I guess u get the idea...

**Comment by harsh-jindal** [comment]
@AakashMallik @fragm3 Made the changes. Please review.

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 1]
```diff
@@ -314,7 +310,7 @@ class ChangePassword extends Component {
           }}
         >
           <form onSubmit={this.changePassword}>
-            <div style={styles.labelStyle}>Current Password</div>
+            <div style={labelStyle}>Current Password</div>
```
Comment (AakashMallik) [inline src/components/Auth/ChangePassword/ChangePassword.react.js]: style={{...labelStyle, color: themeForegroundColor}}

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 2]
```diff
@@ -333,7 +329,7 @@ class ChangePassword extends Component {
               />
             </div>
             <br />
-            <div style={styles.labelStyle}>New Password</div>
+            <div style={labelStyle}>New Password</div>
```
Comment (AakashMallik) [inline src/components/Auth/ChangePassword/ChangePassword.react.js]: Same

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 3]
```diff
@@ -355,7 +351,7 @@ class ChangePassword extends Component {
               <div>{newPasswordStrength}</div>
             </div>
             <br />
-            <div style={styles.labelStyle}>Verify Password</div>
+            <div style={labelStyle}>Verify Password</div>
```
Comment (AakashMallik) [inline src/components/Auth/ChangePassword/ChangePassword.react.js]: Same

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 4]
```diff
@@ -304,6 +292,14 @@ class ChangePassword extends Component {
 
     const PasswordClass = [`is-strength-${newPasswordScore}`];
 
+    const labelStyle = {
```
Comment (AakashMallik) [inline src/components/Auth/ChangePassword/ChangePassword.react.js]: Move this out of the component into the style object. Everything except the ```color: themeForegroundColor``` part

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 5]
```diff
@@ -314,7 +308,9 @@ class ChangePassword extends Component {
           }}
         >
           <form onSubmit={this.changePassword}>
-            <div style={styles.labelStyle}>Current Password</div>
+            <div style={{ ...styles.labelStyle, color: themeForegroundColor }}>
```
Comment (AakashMallik) [inline src/components/Auth/ChangePassword/ChangePassword.react.js]: destructure styles before using

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 6]
```diff
@@ -333,7 +329,9 @@ class ChangePassword extends Component {
               />
             </div>
             <br />
-            <div style={styles.labelStyle}>New Password</div>
+            <div style={{ ...styles.labelStyle, color: themeForegroundColor }}>
```
Comment (AakashMallik) [inline src/components/Auth/ChangePassword/ChangePassword.react.js]: same

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 7]
```diff
@@ -355,7 +353,9 @@ class ChangePassword extends Component {
               <div>{newPasswordStrength}</div>
             </div>
             <br />
-            <div style={styles.labelStyle}>Verify Password</div>
+            <div style={{ ...styles.labelStyle, color: themeForegroundColor }}>
```
Comment (AakashMallik) [inline src/components/Auth/ChangePassword/ChangePassword.react.js]: same

**Pull request merged**